### ISSUE TITLE
.hdr files in pouring dataset

### ISSUE THREAD

**Issue by HKCLynn** [yilundu/nerflow#10]
Hi, author.
Your README.md provide the URL of the pouring dataset. After downloading the dataset I found that all data in .hdr files in the depth file is 255. Maybe there is something wrong with the dataset. And I can't run the  https://github.com/yilundu/nerflow/blob/main/preprocess/depth/run_midas.py. Because it shows **No module named 'models. base_model'** and **No module named 'models. blocks'**. Could you tell me what should I do?

**Comment by yilundu** [comment]
Ohh hmm thats a bit odd -- can you use the code to load the depth? I don't think it should be all 255.

**Comment by HKCLynn** [comment]
Yes, I didn't modify the code. I use the source code to debug. The .hdr file is a 3-channels image. And all data of the image is 255.
![2023-08-17 09-27-28 的屏幕截图](https://github.com/yilundu/nerflow/assets/87703827/4b0f97f4-eebc-42e4-969e-9fcbc97d6e25)
![2023-08-17 09-28-57 的屏幕截图](https://github.com/yilundu/nerflow/assets/87703827/da3dcc4b-42ce-433f-b46c-7a9ec2ca7d5d)

**Comment by HKCLynn** [comment]
![2023-08-18 11-31-59 的屏幕截图](https://github.com/yilundu/nerflow/assets/87703827/2c9b2f0d-c983-4e67-953c-5b4d006ea215)

**Comment by yilundu** [comment]
Hi, could try `depth = imread("r_112_flow0202.hdr", format="HDR-FI")`?
